README.md

```diff
@@ -38,3 +38,15 @@ Exec=gnome-disk-image-mounter %f
 
 Tips:
 - Use `pkexec` to ask for root permissions. Example: `Exec=pkexec chmod +x %U`
+
+## Examples
+- [Send by Email](https://github.com/elementary/mail/blob/master/data/mail-attach.contract)
+- [Print](https://github.com/elementary/pantheon-print/blob/master/data/print.contract)
+- [Send files via Bluetooth](https://github.com/codygarver/os-patch-gnome-bluetooth-xenial/blob/master/debian/gnome-bluetooth.contract)
+- [Compress](https://github.com/codygarver/os-patch-file-roller-xenial/blob/master/data/file-roller-compress.contract)
+- [Extract Here](https://github.com/codygarver/os-patch-file-roller-xenial/blob/master/data/file-roller-extract-here.contract)
+- [Set as Desktop Background](https://github.com/elementary/switchboard-plug-pantheon-shell/blob/master/set-wallpaper-contract/set-wallpaper.contract.in) (will be processed during the compilation process)
```

Comment:
I do not think this should be included as it is not an actual final .contract file.  If it is to be included then perhaps in a separate section with more explanation as to what it is and what the variables are.

Reply:
Good point, I'll remove it for now.